tests/unit_tests/lcls_tools/common/model/gaussian.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import unittest
+from lcls_tools.common.model import gaussian
+
+
+class TestGaussianFits(unittest.TestCase):
```

Comment:
We should add a test that pulls a defined set of data from a file that doesn't change. It's not needed for this PR, just a note.